src/main/resources/assets/openblocks/lang/de_de.lang

```diff
@@ -222,91 +222,91 @@ item.openblocks.sponge_on_a_stick.name=Schwamm
 item.openblocks.sponge_on_a_stick.description=Der Schwamm am Stiel ist ein Werkzeug zum Enfernen von Flüssigkeiten. Funktioniert wie der Schwamm, ist aber mobil und wird mit der Zeit aufgebraucht.
 
 item.openblocks.epic_eraser.name=Geschichtsveränderndes Radiergummi
-item.openblocks.epic_eraser.description=Jedes Item erzählt eine Geschichte...\n\nIst das nicht eigentlich saublöd?\n Mit diesem Item kannst du jegliche Spuren epischer Abenteuer, die dein Kram im Laufe der Jahrtausende erlebt hat, vollständig vernichten. Dann wird auch keiner Fragen stellen, wenn du ihn auf dem Schwarzmarkt vertickst.
+item.openblocks.epic_eraser.description=Jeder Gegenstand erzählt eine Geschichte...\n\nIst das nicht eigentlich saublöd?\n Mit diesen Gegenstand kannst du jegliche Spuren epischer Abenteuer, die dein Kram im Laufe der Jahrtausende erlebt hat, vollständig vernichten. Dann wird auch keiner Fragen stellen, wenn du ihn auf dem Schwarzmarkt vertickst.
 
-tile.openblocks.cannon.name=Item-Kanone
-tile.openblocks.cannon.description=Wenn ein Redstone-Signal anliegt, wird die Item-Kannone Items, die es in einem benachbarten Inventar findet, schießen. Benutze den 'Zeiger', um die Kannone auszurichten.
+tile.openblocks.cannon.name=Gegenstands-Kanone
+tile.openblocks.cannon.description=Wenn ein Redstone-Signal anliegt, wird die Gegenstands-Kannone Gegenstände, die es in einem benachbarten Inventar findet, schießen. Benutze den 'Zeiger', um die Kannone auszurichten.
 
 tile.openblocks.rope_ladder.name=Strickleiter
 tile.openblocks.rope_ladder.description=Wenn du eine Strickleiter platzierst, wird sie sich bis zum Boden ausrollen.
 
 tile.openblocks.sprinkler.name=Rasensprenger
-tile.openblocks.sprinkler.description=Der Rasensprenger hilft dir, Pflanzen bis zu 4 Blöcke weit weg zu düngen. Platziere ihn einfach über einem OpenBlocks-Tank, der mit Wasser gefüllt ist! Du kannst auch Knochenmehl für noch schnelleres Wachstum hinzufügen.\nEr zählt auch als Wasser-Quellblock!
+tile.openblocks.sprinkler.description=Der Rasensprenger hilft dir, Pflanzen bis zu 4 Blöcke weit weg, zu düngen. Platziere ihn einfach über einem OpenBlocks-Tank, der mit Wasser gefüllt ist! Du kannst auch Knochenmehl für noch schnelleres Wachstum hinzufügen.\nEr zählt auch als Wasser-Quellblock!
```

Comment:
That comma does not belong there.

Reply:
The first time I read it: "weg zu düngen" xD

Reply:
Doesn't change the fact that that comma is placed wrong.